068-persistent-bottom-nav-bar/lib/ui/views/posts/posts_viewmodel.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:persistent_bottom_nav_bar/app/app.locator.dart';
+import 'package:persistent_bottom_nav_bar/app/router_ids.dart';
+import 'package:persistent_bottom_nav_bar/models/post_model.dart';
+import 'package:persistent_bottom_nav_bar/ui/views/post_details/post_details_view.dart';
+import 'package:stacked/stacked.dart';
+import 'package:stacked_services/stacked_services.dart';
+
+class PostsViewModel extends BaseViewModel {
+  final _navigationService = locator<NavigationService>();
+  List<Post> get posts => postsData;
+
+  void onPostTap(Post post) {
+    _navigationService.navigateWithTransition(
+      PostDetailsView(post: post),
```

Comment:
This doesn't account for nested named routing which is what would be required. We can't be mixing the routing. 

I'd lke to see this as apart of the navigator